packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -342,7 +343,7 @@ export function useSelectableCollection(options: AriaSelectableCollectionOptions
       scrollRef.current.scrollLeft = scrollPos.current.left;
     }
 
-    if (!isVirtualized && manager.focusedKey != null) {
+    if (manager.focusedKey != null) {
```

Comment:
Was removing !isVirtualized intentional here?

Reply:
yeah we centralized the logic between both virtualized and non-virtualized collections so now they are both handled here. was there a concern or just curious?